pkg/controller/certificaterequest/certificaterequest_controller_test.go

```diff
@@ -267,18 +272,31 @@ func TestReconcile(t *testing.T) {
 			expectError: false,
 		},
 		{
-			name: "reqeusts a new cert",
+			name: "handles multiple ingresses and cert secrets",
 			clientObjects: func() []runtime.Object {
 				cr := &certmanv1alpha1.CertificateRequest{}
 				cr.TypeMeta = certRequest.TypeMeta
 				cr.ObjectMeta = certRequest.ObjectMeta
 				//cr.ObjectMeta.OwnerReferences = []metav1.OwnerReference{} // this should already be set to actual owner refs by ^
 				cr.Spec = certRequest.Spec
 				// cr.Status = certRequest.Status // need to test that it sets the status
+				cr.Spec.CertificateSecret.Name = "ocm-cert-bundle-secret"
 
 				// generate an leclient using a mock acme client
 				newClientSecret := testLESecret
 				newClientSecret.Data["account-url"] = []byte("proto://use.mock.acme.client")
+				clusterDeploymentComplete.Spec.CertificateBundles = append(clusterDeploymentComplete.Spec.CertificateBundles, hivev1.CertificateBundleSpec{
+					Name:     "ocm-cert-bundle-secret",
```

Comment:
Can we omit anything related to "ocm"? I was just using "ocm" as an example of an arbitrary certificate and secret name.

Reply:
gotcha, i've got a new commit with that